### ISSUE TITLE
Issue with sendmail (busybox v1.28.4)

### ISSUE THREAD

**Issue by daohoangson** [docker-library/php#756]
I have encountered this bug in sendmail with `php:7.2.12-alpine3.8`, reproduce steps:

We have this in /usr/local/etc/php/conf.d/relay.ini

```
sendmail_path=/usr/sbin/sendmail -S relay.domain.com -t -i
```

And this simple PHP script:

```php
<?php

mail('Name <address@domain.com>', 'Foo', 'Bar', '', "-f 'php@domain.com'");

# this works (recipient without name)
# mail('address@domain.com', 'Foo', 'Bar', '', "-f 'php@domain.com'");
```

This is the log from postfix:

```
postfix/smtpd[94]: connect from php[192.168.16.2]
postfix/smtpd[94]: disconnect from php[192.168.16.2] ehlo=1 mail=1 data=0/1 quit=1 commands=3/4
```

Basically the MAIL commands failed. I have tracked down the bug to this commit [Aug 2017](https://git.busybox.net/busybox/commit/mailutils/sendmail.c?id=20077c1429). Which has been fixed by this commit [July 2018](https://git.busybox.net/busybox/commit/mailutils/sendmail.c?id=6f09785b7e4a6cc7a9ede623d51aa8f6dc5e9a03). Unfortunately, busybox v1.28.4 (which ships with `alpine:3.8`) latest commit is from [May 2018](https://git.busybox.net/busybox/commit/?h=1_28_4) so the fix didn't make the cut... The latest working php image is `php:7.2.11-alpine3.7` (which has busybox v1.27.2 -- [it doesn't have the buggy commit](https://git.busybox.net/busybox/log/?h=1_27_2)).

There are a few options for me:

1. Ask alpine image maintainer nicely so they can bump version of busybox (latest is 1.29.3 according to https://busybox.net/)
2. Ask php image maintainer nicely to use alpine:3.7 as base image -- probably won't happen 😝
3. Patch the external script (not developed by us) to drop name from recipient address -- already done

What do you advice?

**Comment by yosifkit** [comment]
1. why not just use the [`postfix` package](https://pkgs.alpinelinux.org/package/v3.8/main/x86_64/postfix) instead of the `postfix` from `busybox`?
2. the alpine 3.7 based images are still available for `docker pull`; they are no longer updated and were dropped intentionally (https://github.com/docker-library/php/pull/730).

Also, Alpine 3.9 should be coming this month so we will have those images soon.

**Comment by daohoangson** [comment]
1 I was using `sendmail` from `busybox`, not `postfix`, the bug is in `sendmail@busybox`. Maybe you meant using `sendmail` from `postfix` packge? I haven't thought of that! 
2 Yes, I understand alpine:3.7 images are still available up to php:7.2.11. It's good to know we are moving to alpine:3.9 soon though.
